server/room-battle.js

```diff
@@ -559,6 +559,7 @@ class RoomBattle {
 		request.choice = choice;
 
 		this.stream.write(`>${player.slot} ${choice}`);
+		this.allowTie = ['', ''];
```

Comment:
This resets ``allowTie`` too often. It needs to be reset when we get a new request from the sim.